crates/solidity/outputs/rust/lib/src/generated/kinds.rs

```diff
@@ -5,107 +5,114 @@ use strum_macros::*;
 
 #[derive(Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord, Serialize)]
 pub enum TokenKind {
-    Abicoder,
-    Abstract,
+    AbicoderKeyword,
+    AbstractKeyword,
     Address,
```

Comment:
I think there are two `terminal: "address"` left in `crates/solidity/inputs/schema/grammar/topics/05-expressions/01-base-expressions.yml`

Reply:
done